docs/src/content/pages/fields/object.mdoc

```diff
@@ -1,11 +1,13 @@
 ---
 title: Object field
 ---
-The `object` field lets you structure complex object schemas that can contain any other field types.
+The `object` field is used to create complex object schemas that can contain any other fields. 
 
-## Usage example
+It's particularly useful in conjunction with the [array](/docs/fields/array) field (that only accepts one single field) when you need multiple fields for each item in the array.
```

Comment:
It is also useful for conditional and blocks fields, should we add those examples there too, or is it getting to wordy?

Reply:
Have tweaked now, let me know what you think!